types/hooks/screens/useAnonymousChannelListScreenHook.types.ts

```diff
@@ -1,10 +1,11 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseAnonymousChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `UseAnonymousChannelListScreenHook` interface are good. The `goToContactScreen` function now accepts a parameter of type `ContactScreenPayload`, which provides more context when navigating to the Contact Screen. This change improves the flexibility and reusability of the hook.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->